scraper/src/scraper/providers/anthropic.py

```
@@ -12,12 +13,33 @@
+# Claude Sonnet 5 促進価格の最終適用日（2026-08-31 まで $2/$10、以降 $3/$15）。
+_SONNET_5_PROMO_UNTIL = datetime.date(2026, 8, 31)
+
+
+def _sonnet_5_fallback(today: datetime.date | None = None) -> tuple[float, float]:
+    """Claude Sonnet 5 のフォールバック価格を適用日で切り替える。
+
+    Sonnet 5 はライブ抽出対象外（フォールバック固定）のため、促進価格の
+    期限をここで表現する。2026-08-31 まで促進価格 $2/$10、2026-09-01 以降は
+    恒久価格 $3/$15 を返す（_SUB_JA / _SUB_EN の説明と一致させる）。
+    """
+    day = today or datetime.date.today()
```

Ruff (0.15.21)

[warning] `datetime.date.today()` used

(DTZ011)